source/docs/appendices/migration-guides.md

```diff
@@ -7,6 +7,217 @@ This area will contain the Migration steps to follow for upgrading your store to
+
+It allows developer to choose the version of `date-fns` they prefer and reduce the bundle size. Please ensure you are not using `date-fns` in your application, or install it in your project. The version in Front-Commerce was 1.30.1, and the latest available is 2.14.0.
+
+For backwards compatibility, you can run:
+```shell
+npm install --save date-fns@1.30.1
+```
+
+> **Logging format update:** a timestamp is now automatically added to log messages as the `timestamp` field of each entry (in GMT time). `date` fields were removed. You can update your custom logs to do so too.
+
+See [this commit](https://gitlab.com/front-commerce/front-commerce/-/commit/bf9f9ace04bc24a7391fe36c267dd94857663db0) and [this one](https://gitlab.com/front-commerce/front-commerce/-/commit/60c31720fe3f113961555c493417b8c8bd45509b) for an overview of those changes.
+
+### Make linting pass again
+
+Dependencies were updated, a first step would be to automatically update as much as your codebase as possible to ensure it matches the latest conventions and good practices.
+You can run:
+
+```shell
+npm run lint -- --fix
+```
+
+Below are the main updates to be aware of.
+
+#### Prettier updated to 2.x
+
+We've updated Prettier to its latest version (1.x -> 2.x). It will not cause any issues to your application, but may break snapshot tests. Your code will have to be reformatted with the latest Prettier version.
+
+It introduced trailing commas, parents for single param arrow functions and other changes. You can read [Prettier 2.0 "2020" blog post](https://prettier.io/blog/2020/03/21/2.0.0.html) for further details.
+
+#### ESLint and rules updated to 7.x
+
+ESLint (and related rules) have been updated to their latest version. Please ensure your code match the latest rules if your app uses Front-Commerce's default rules. `lint-staged` has also been updated from 9.x to 10.x and requires node 10.13+.
+
+See [Migrating to v7.0.0](https://eslint.org/docs/user-guide/migrating-to-7.0.0) for further details.
+
+### ElasticSearch related changes
+
+#### `@elastic/elasticsearch` must be installed
+
+Recent versions of the official [`@elastic/elasticsearch`](https://www.npmjs.com/package/@elastic/elasticsearch) client are versioned against the corresponding ElasticSearch version they are compatible with. We decided to make it a [peer-dependency](https://nodejs.org/es/blog/npm/peer-dependencies/) of `front-commerce`'s package instead of being a direct dependency to allow developers to choose the exact version they need.
+
+You must install it in your application, with the version matching your server version. For now, only 6.x is supported.
+
+```shell
+npm install --save @elastic/elasticsearch@6
+```
+
+**This library had several breaking changes.** The most important one is that `client.search` now returns an object instead of the body.
```

Comment:
I'd add a reassuring message telling the users that this will affect them only if they changed how products were retrieved from ES. If they only use configurations available in `website.js` they don't have to worry about this update.